src/libultra_code/sins.c

```diff
@@ -1,3 +1,20 @@
-#include "libultra_internal.h"
+#include <ultra64.h>
+#include "sintable.c"
```

Comment:
I think we should keep header and C files includes separate since they are a very different thing.
```suggestion
#include <ultra64.h>

#include "sintable.c"
```